libkbfs/folder_branch_ops.go

```diff
@@ -165,6 +170,17 @@ func (bl *blockLock) DoRUnlockedIfPossible(lState *lockState, f func(*lockState)
 	f(lState)
 }
 
+// headTrustStatus marks whether the head is from a trusted or
+// untrusted source. When rekeying we get the fbo by folder
+// id and do not check the tlf id, thus resulting in untrusted
+// fbo heads.
+type headTrustStatus bool
```

Comment:
i don't think this is desirable, since then `headTrustStatus` can be used in an `if` statement without comparing it to a constant value. int would be better.